### ISSUE TITLE
Generate datasets of triangles and squares

### ISSUE THREAD

**Issue by elboyran** [NLeSC/XAI#11]
Different directions to study what is used for classification.

- Binary

- [x] Load CSV files with images and with labels and convert to MAT arrays
- [x] Fix shape labels, 0 = square, 2 = triangle
- [x] Split into train, test and validation sets

- Gray

- [x] Load CSV files with images and with labels and convert to MAT arrays
- [x] Split into train, test and validation sets

- [x] Load CSV files with bands and convert to MAT arrays
- [x] Get and save the bands for the test set

**Comment by elboyran** [comment]
```
images = csvread('Polygons.dat'); labels = csvread('Class.dat');
save('TrianglesAndSquares_labels_all.mat', 'labels');
save('TrianglesAndSquares_images_all.mat', 'images');
```

**Comment by elboyran** [comment]
The gray data-set is too large to store on github.

**Comment by elboyran** [comment]
```
bands = csvread('Bands.dat');
save('TrianglesAndSquares_bands_all.mat','bands');
train_lim = 50000;
test_lim = 80000;
test_bands = bands(train_lim+1:test_lim,:);
save('TrianglesAndSquares_bands_test.mat','test_bands');
```
